apps/chat/src/pages/index.tsx

```diff
@@ -229,9 +238,8 @@ export default function Home({ initialState }: HomeProps) {
             <div className="flex size-full flex-col sm:pt-0">
               {enabledFeatures.has(Feature.Header) && <Header />}
               <div className="flex w-full grow overflow-auto">
-                {enabledFeatures.has(Feature.ConversationsSection) && (
-                  <Chatbar />
-                )}
+                {enabledFeatures.has(Feature.ConversationsSection) &&
```

Comment:
Maybe it's will be easier to just set in getServerSideProps needed features than add additional iofs on all levels of rendering?

Reply:
@mikitabut , `window is not defined` into `getServerSideProps`